src/presenters/rel-next-presenter.php

```diff
@@ -56,4 +61,15 @@ private function filter() {
 		 */
 		return (string) \trim( \apply_filters( 'wpseo_adjacent_rel_url', $this->presentation->rel_next, 'next', $this->presentation ) );
 	}
+
+	/**
+	 * Escaped the output.
+	 *
+	 * @param string $value The value.
+	 *
+	 * @return string The escaped value.
+	 */
+	protected function escape( $value ) {
```

Comment:
If we add this to the abstract based on the `$escaping` class attribute, this isn't needed, right?